lib/matplotlib/axes/_axes.py

```diff
@@ -2469,105 +2469,107 @@ def stem(self, *args, **kwargs):
         return stem_container
 
     @_preprocess_data(replace_names=['x', 'explode', 'labels', 'colors'],
-                         label_namer=None)
+                      label_namer=None)
     def pie(self, x, explode=None, labels=None, colors=None,
             autopct=None, pctdistance=0.6, shadow=False, labeldistance=1.1,
             startangle=None, radius=None, counterclock=True,
             wedgeprops=None, textprops=None, center=(0, 0),
             frame=False, rotatelabels=False):
-        r"""
-        Plot a pie chart.
+        r"""Plot a pie chart.
```

Comment:
I don't think so, but if everything breaks when you remove the raw string, then it needs to be a raw string :p